Library/Formula/dcd.rb

```diff
@@ -0,0 +1,13 @@
+require "formula"
+
+class Dcd < Formula
+  homepage "https://github.com/Hackerpilot/DCD"
+  url "https://github.com/Hackerpilot/DCD.git", :using => :git, :tag => "v0.5.1"
+
+  depends_on "dmd" => :build
+
+  def install
+    system "make"
+    bin.install "bin/dcd-client", "bin/dcd-server"
```

Comment:
No ` make install `?

` bin.install Dir["bin/*"] ` would work here too if there's nothing else in that directory that we need to worry about.